backend/server/rhnServer/server_hardware.py

```diff
@@ -1231,5 +1231,5 @@ def reload_hardware_byid(self, sysid):
 
 
 For more information on generating a new machine_id for traditional systems refer to:
-https://www.suse.com/documentation/suse-manager-3/book_suma_best_practices_31/data/bp_chapt_suma3_troubleshooting_registering_cloned_traditional_systems.html
+https://documentation.suse.com/external-tree/en-us/suma/4.0/suse-manager/administration/tshoot-registerclones.html
```

Comment:
Mmmmmm, shouldn't this point to:
https://www.uyuni-project.org/uyuni-docs/suse-manager/suse-manager/administration/tshoot-registerclones.html if the server is Uyuni?